### ISSUE TITLE
Repair Zipkin Micrometer Tracing

### ISSUE THREAD

**Issue by arey** [spring-petclinic/spring-petclinic-microservices#223]
It seems that since the upgrade to Spring Cloud 2022.0, the OpenZipkin server is no longer receiving the traces. 
We should use Micrometer to trace your Spring Boot app: https://springbootlearning.medium.com/using-micrometer-to-trace-your-spring-boot-app-1fe6ff9982ae

**Comment by MarianConstantinMarica** [comment]
Can I work on this issue?

**Comment by arey** [comment]
Yes @MarianConstantinMarica you can work on this issue.  Thank you for the proposal.

**Comment by arey** [comment]
@dgriffiths-undo you may try the previous version: https://github.com/spring-petclinic/spring-petclinic-microservices/tree/v2.6.7

**Comment by dgriffiths-undo** [comment]
Ok that works, thanks!

**Comment by MarianConstantinMarica** [comment]
I sent the PR

**Comment by arey** [comment]
Thanks @MarianConstantinMarica I've just posted a single comment. Once merged, @dgriffiths-undo could you please test it?

**Comment by arey** [comment]
@marcingrzejszczak I saw you are working on Micrometer. Thus could you have a look on this PR please?
